public/views/create_bug.tpl

```diff
@@ -1,21 +1,23 @@
-<form id="createBug">
+<div>
 
   <header data-version="1">
-    <a class="headerBtn close hidden"></a>
+    <a class="headerBtn close"></a>
     <h2>File a Bug</h2>
-    <a href="/logout/" class="headerBtn logout" hidden>logout</a>
+    <a href="/logout/" class="headerBtn logout" hidden>Logout</a>
   </header>
 
-  <input type="text" placeholder="Title" id="summary" />
-  <input list="components" id="component" value="Gaia::Feedback" disabled hidden />
-  <textarea placeholder="Description" id="description" ></textarea>
+  <form id="createBug">
+    <input type="text" placeholder="Title" id="summary" />
+    <input list="components" id="component" value="Gaia::Feedback" disabled hidden />
+    <textarea placeholder="Description" id="description" ></textarea>
 
-  <span class="btn-file attachBtn attach">
-    <input type="file" multiple />
-    Add an Attachment
-  </span>
-  <ul class="attachments"></ul>
+    <span class="btn-file attachBtn attach">
+      <input type="file" multiple />
+      Add an Attachment
```

Comment:
Yeh l10n needs to be done but given we havent got a scheduled release based off master for a long time I think it may be possible to merge before hand, but we can deal that when it comes to merge time